app/api/__init__.py

```diff
@@ -398,3 +401,7 @@
 api.route(OrderRelationship, 'order_event', '/orders/<identifier>/relationships/event')
 api.route(OrderRelationship, 'order_marketer', '/orders/<identifier>/relationships/marketer')
 api.route(OrderRelationship, 'order_discount', '/orders/<identifier>/relationships/discount-code')
+
+# Event Analytics API
+api.route(EventAnalyticsGeneralDetail, 'event_analytics_general_detail', '/analytics/events/<int:id>/general',
```

Comment:
But according to https://flask-rest-jsonapi.readthedocs.io/en/latest/resource_manager.html#resourcerelationship , You can change the datalayer used. So, it should be possible to use a custom data layer and mimic an sqlalchemy relationship.